docs/samples/explanation/aif/germancredit/bias.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: serving.kubeflow.org/v1alpha2
```

Comment:
Perhaps better to us the v1beta1 format since v1alpha2 will be deprecated in the v0.6.0 release.